pkgs/development/python-modules/pwinput/default.nix

```diff
@@ -0,0 +1,28 @@
+{
+  buildPythonPackage,
+  fetchPypi,
+  setuptools,
+  lib,
+}:
+
+buildPythonPackage rec {
+  pname = "pwinput";
+  version = "1.0.3";
+
+  src = fetchPypi {
+    inherit pname version;
```

Comment:
```suggestion
    inherit version;
    pname = "pwinput";
```

Reply:
I don't think we should reuse `pname` for anything else.

Reply:
Alright